modules/server/src/network/resolvers/index.ts

```diff
@@ -12,29 +11,33 @@ type NetworkSearchRoot = {
 
 /**
  * Create GQL resolvers.
+ *
+ * It's important to have both remote connection data and aggregations under a single field
+ * as remote connection data is dependant on aggregations query
+ *
  * @param networkConfigsWithSchemas
  * @param networkFieldTypes
  * @returns
  */
 export const createResolvers = (configs: NetworkAggregationConfig[]) => {
 	return {
 		Query: {
-			nodes: async () => await resolveRemoteConnectionNodes(configs),
-			aggregations: async (
+			network: async (
 				parent: NetworkSearchRoot,
 				args: {},
 				context: unknown,
 				info: GraphQLResolveInfo,
 			) => {
-				const rootQueryFields = getRootFields(info);
-				const networkQueries = createNetworkQueries(configs, rootQueryFields);
-				const networkResults = await queryConnections(networkQueries);
-				// Aggregate queried data
-				const resolvedResults = resolveAggregations(networkResults, rootQueryFields);
-				// TODO: format to well defined response object createResponse(resolvedResults) jon success/failure, conform to schema shape etc
-				const response = resolvedResults.reduce((response, currentField) => {
-					return { ...response, ...{ [currentField.fieldName]: { ...currentField.aggregation } } };
-				}, {});
+				const { requestedAggregations } = getRequestedFields(info);
+				const networkQueries = createNetworkQueries(configs, requestedAggregations);
+
+				// Query remote connections and aggregate results
+				const { aggregationResults, nodeInfo } = await aggregationPipeline(
+					networkQueries,
+					requestedAggregations,
+				);
+				const response = createResponse({ aggregationResults, nodeInfo });
+				console.log('resp', response);
```

Comment:
This looks like test logging

Reply:
thank you - updated - https://github.com/overture-stack/arranger/pull/887/commits/9e3f631e7cdab75e46c020e439b1a29284a7aeaf